openshift/release/release-notes.md

```diff
@@ -0,0 +1,376 @@
+
+  Fix an issue where the PipelineRun controller could create duplicate TaskRuns if a Pipeline's or PipelineRun's labels are changed while the PipelineRun is running
+
+* :bug: Consider not-found pod as permanent error when taskrun is done ([#3542](https://github.com/tektoncd/pipeline/pull/3542))
+
+  Fix an issue where the taskrun controller would continue reconciling completed taskruns if pods in case of evicted pods
+
+* :bug: optimize cycle detection logic in dag ([#3539](https://github.com/tektoncd/pipeline/pull/3539))
+
+* :bug: Fix script step on CRI-O runtime ([#3526](https://github.com/tektoncd/pipeline/pull/3526))
+## Misc
+* :hammer: Do not set the gitconfig option globally ([#3574](https://github.com/tektoncd/pipeline/pull/3574))
+
+* :hammer: fix commen typo ([#3546](https://github.com/tektoncd/pipeline/pull/3546))
+
+* :hammer: deleting redundant test file ([#3556](https://github.com/tektoncd/pipeline/pull/3556))
+
+* :hammer: Use logstream in the duplicate_test ([#3530](https://github.com/tektoncd/pipeline/pull/3530))
+
+* :hammer: pkg/apis: rename {Task,Pipeline}Interface into *Object ([#3589](https://github.com/tektoncd/pipeline/pull/3589))
+
+* :hammer: retrieving deps outside of dag.build ([#3583](https://github.com/tektoncd/pipeline/pull/3583))
+
+* :hammer: pkg/apis: remove duplicate substitution code. ([#3579](https://github.com/tektoncd/pipeline/pull/3579))
+
+* :hammer: pkg/apis: unexport ApplyContainerReplacements ([#3578](https://github.com/tektoncd/pipeline/pull/3578))
+
+* :hammer: pkg/apis: use pod.Template instead of v1beta1.PodTemplate ([#3577](https://github.com/tektoncd/pipeline/pull/3577))
+
+* :hammer: tracker: use TrackReference instead of deprecated Track function ([#3576](https://github.com/tektoncd/pipeline/pull/3576))
+
+* :hammer: pkg/apis: refactor GetTaskRunSpecs function ([#3575](https://github.com/tektoncd/pipeline/pull/3575))
+
+* :hammer: pkg/apis: remove duplicate MergeStepsWithStepTemplate function ([#3572](https://github.com/tektoncd/pipeline/pull/3572))
+
+* :hammer: Make examples runnable with kubectl rabbit2 ([#3564](https://github.com/tektoncd/pipeline/pull/3564))
+
+* :hammer: Refresh list of s390x excluded test ([#3527](https://github.com/tektoncd/pipeline/pull/3527))
+
+* :hammer: Add some tests for pkg/git/git.go ([#3523](https://github.com/tektoncd/pipeline/pull/3523))
+
+* :hammer: Use dogfooding skopeo image for e2e and examples tests ([#3519](https://github.com/tektoncd/pipeline/pull/3519))
+
+* :hammer: Use dogfooding buildx image for multi-arch builds ([#3514](https://github.com/tektoncd/pipeline/pull/3514))
+
+* :hammer: pkg/apis tests cleanup on duplicate test name nail_care ([#3495](https://github.com/tektoncd/pipeline/pull/3495))
+
+* :hammer: Update plumbing dep to update google/go-licenses ([#3422](https://github.com/tektoncd/pipeline/pull/3422))
+
+## Docs
```

Comment:
Do we usually keep the docs part ?

Reply:
i saw last release notes it was there

It shows what all things we do documentation so may be we can keep

But i am okay to remove

Reply:
We do not document this in downstream, so we can remove this.